rust/tools/authorization-logic/test_inputs/hyphenIDs

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2022 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+.decl ourFavoritePublicKeyIs(k : Key)
+.decl ourFavoriteTrustedBuilderIs(b : Builder)
+.decl variablesInsteadOfConstantsWithHyhensAreOKToo(v : TestVariable)
+"OakTeam" says {
+    ourFavoritePublicKeyIs(
+"""-----BEGIN PUBLIC KEY-----
+MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAErnyr7XY6gxXPnqRGYEpv5lDsbBeX
+izVFLckaF/cbgBuzx7kfmf1qi2j+3DDOQYgISQOj9/LYt4c1cxbt+XtG4g==
+-----END PUBLIC KEY-----""").
+    ourFavoriteTrustedBuilderIs("https://github.com/project-oak/transparent-release").
+    ourFavoriteTrustedBuilderIs("https://github.com/Attestations/GitHubHostedActions@v1").
```

Comment:
Is sanitizing `"` from filepaths tested here? I'm not seeing any `"` in the middle of these files. Maybe this is supposed to address the `"`s at the beginning and end?

Reply:
Sanitizing `"` out of filepaths was actually something this code was doing before this PR. I just refactored it here (for example, see the old line 230 where the `replace` was being done where the filename was parsed. It was also doing this `replace` in all the the other spots where a filename was parsed. ) 

So this new test does not test sanitization of filepath names, but other pre-existing tests already did like [this one](https://github.com/google-research/raksha/blob/main/rust/tools/authorization-logic/test_inputs/quotesInExports), so this change must not have caused a regression.